php73-pecl-redis.spec

```
@@ -28,17 +28,21 @@
 
 Summary:       Extension for communicating with the Redis key-value store
 Name:          %{php}-pecl-%{pecl_name}
-Version:       4.3.0
-Release:       3%{?dist}
+Version:       5.0.2
+Release:       1%{?dist}
 Source0:       https://pecl.php.net/get/%{pecl_name}-%{version}.tgz
 License:       PHP
 URL:           https://pecl.php.net/package/%{pecl_name}
 
 BuildRequires: gcc
 BuildRequires: %{php}-devel
```

```suggestion
BuildRequires: %{php}-devel > 7
```

It's slightly redundant, but the upstream PECL module dropped support for PHP5 with this release, so on the off chance someone tries to adapt this spec file for php56u (or older) this will be an additional hint about why it won't work.